lms/envs/common.py

```diff
@@ -1328,6 +1328,20 @@
         ],
         'output_filename': 'css/lms-style-xmodule-annotations.css',
     },
+    'style-certificates': {
+        'source_filenames': [
+            'sass/certificates/main-ltr.css',
```

Comment:
I thought the compiled css was checked into the css directory.  Why are we referencing it from the sass directory.